### PR TITLE
Microsoft Live Connect integration

### PR DESCRIPTION
**Pull request by ant30**
I Added the Microsft Live Connect auth backend to pyramid_sna

### CODE REVIEW

File: pyramid_sna/oauth2.py [hunk 1]
```diff
@@ -78,6 +78,6 @@ def get_user_info(info_uri, access_token):
     response = requests.get(info_uri, headers=headers)
 
     if response.status_code != 200:
-        return HTTPUnauthorized(response.text)
+        raise HTTPUnauthorized(response.text)
```
Comment (lorenzogil) [inline pyramid_sna/oauth2.py:81]: Can you explain why is it better to raise the HTTPUnauthorized response instead of returning it?

**Comment by lorenzogil** [comment]
This contribution looks really great. Thanks a lot for adding LiveConnect support to pyramid_sna.

Other than the small question about raising HTTPUnauthorized response, everything looks good to me. So once we deal with that small thing I'll merge this into master and do a new release.

**Comment by ant30** [comment]
Hi Lorenzo,

The raise and return have the same effect over the request. But the thing
is this: if the http exception is not directly in the function or view
method, then is more readable the raise way.  If we consider sna callbacks
as subroutines, you can read what the pyramid cookbook tells about the
exceptions.

http://docs.pylonsproject.org/projects/pyramid_cookbook/en/latest/pylons/exceptions.html

But if you want I can revert that change which was made in May.

Kind regards,
Antonio.

**Comment by lorenzogil** [comment]
Ok, I don't have a strong opinion over this and your reason is good enough, so I'll keep the raise statement.

**Pull request merged**